lib/Fmt_ast.ml

```diff
@@ -557,6 +566,26 @@ let rec fmt_extension c ctx key (ext, pld) =
   | _, _, PSig [({psig_desc= Psig_type _; _} as si)], (Pld _ | Sig _ | Top)
     ->
       fmt_signature_item c ~ext (sub_sig ~ctx si)
+  (* Quoted extensions (since ocaml 4.11) {%ext delim|...|delim}. Comments
+     and attributes are not allowed by the parser. *)
+  | ( ("%" | "%%")
+    , ext
+    , PStr
+        [ { pstr_desc=
+              Pstr_eval
+                ( { pexp_desc= Pexp_constant (Pconst_string (str, _, delim))
```

Comment:
The second argument of `Pconst_string` is a location that has to be considered for comments (with an assert like the others)

Reply:
done